test/integration/test_sqlite_backend.py

```diff
@@ -58,8 +58,8 @@ async def test_keys_values_size(cache_client):
         await cache_client.write(k, v)
 
     assert await cache_client.size() == len(test_data)
-    assert await cache_client.keys() == list(test_data.keys())
-    assert await cache_client.values() == list(test_data.values())
+    assert [k async for k in cache_client.keys()] == list(test_data.keys())
+    assert [v async for v in cache_client.values()] == list(test_data.values())
```

Comment:
I'm not sure how deterministic the order is for various backends. For dynamodb, I wrapped these in set() and used set comprehension to avoid out-of-order diffs. alternatively can sort. But I think if there was a consistent pattern across the tests, both for consistency's sake and to illustrate that the order here is not guaranteed, that might be cool.

Reply:
Good call. I'll wrap those in `set`.

As for consistency/reducing copy-pasted tests, in the future I'll probably add a base test that will test all the common public methods, and will be parametrized to run for all backend classes.

Reply:
Tests have been updated.